__tests__/html/activityStatusTelemetry.sendFiles.attachmentUrl.sendingToSent.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <link href="/assets/index.css" rel="stylesheet" type="text/css" />
+    <script crossorigin="anonymous" src="/test-harness.js"></script>
+    <script crossorigin="anonymous" src="/test-page-object.js"></script>
+    <script crossorigin="anonymous" src="/__dist__/webchat-es5.js"></script>
+  </head>
+  <body>
+    <div id="webchat"></div>
+    <script>
+      run(async function () {
+        const testAttachmentActivity = {
+          type: 'message',
+          id: 'CONVERSATION_ID-o|00000',
+          timestamp: '2000-01-23T12:34:56.12345Z',
+          channelId: 'directline',
+          from: {
+            id: 'webchat-mockbot',
+            name: 'webchat-mockbot'
+          },
+          conversation: {
+            id: 'CONVERSATION_ID-o'
+          },
+          locale: 'en-US',
+          attachments: [
+            {
+              contentUrl: 'https://bing.com/',
+              contentType: 'text/plain',
+              name: 'test.txt'
+            }
+          ]
+        }
+        const store = testHelpers.createStore();
+
+        const directLine = testHelpers.createDirectLineEmulator(store);
+
+        const telemetryEventTracking = [];
+        WebChat.renderWebChat(
+          {
+            directLine,
+            store,
+            onTelemetry: (event) => {
+              telemetryEventTracking.push(event);
+            },
+            styleOptions: {
+              sendTimeout: 5000
+            }
+          },
+          document.getElementById('webchat')
+        );
+
+        await pageConditions.uiConnected();
+
+        const sendMessage1stAttempt = await directLine.emulateOutgoingActivity(testAttachmentActivity);
+        await sendMessage1stAttempt.echoBack();
+        sendMessage1stAttempt.resolvePostActivity();
```

Comment:
Nit. Notes:
- New line after `const` block
- `resolveAll()` is a handy version of `echoBack()` + `resolvePostActivity()`

```suggestion
        // GIVEN: An outgoing activity has sent successfully.
        const sendMessageFirstAttempt = await directLine.emulateOutgoingActivity(testAttachmentActivity);

        await sendMessageFirstAttempt.resolveAll();
```

Reply:
updated